lib/curated/utils/md_formating.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+module Utils
+  extend self
+  def ruby_example_code(text)
+    (+'```ruby') << "\n#{text}\n```"
+  end
```

Comment:
What do you think about renaming this to `code_block(language, text=nil, &block)` or something?